recipes/nnlojet/recipe.yaml

```
@@ -0,0 +1,102 @@
+context:
+  name: "nnlojet"
+  version: "1.0.0"
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://nnlojet.hepforge.org/nnlojet-v${{ version }}.tar.gz
+  sha256: b02013922387cf37745487d23b271eaae257c8af76bb3cca6bbf885dd8139e74
+
+build:
```

```suggestion
build:
  detect_binary_files_with_prefix: true
```

This might be needed?